sdk/core/azure-core-amqp/src/models/message_source.cpp

```diff
@@ -24,7 +27,115 @@ namespace Azure { namespace Core { namespace _internal {
 
 namespace Azure { namespace Core { namespace Amqp { namespace Models { namespace _internal {
 
+  MessageSource::~MessageSource(){};
```

Comment:
super nit, really a matter of personal preference, if you also like it. This would apply to all such destructor, only commenting on the first one.
```suggestion
  MessageSource::~MessageSource() = default;
```

Reply:
I didn't realize you could declare an implementation as "default", only a declaration.